utils.py

```diff
@@ -7,3 +7,10 @@ def emit_event(controller, name, **kwargs):
     event_name = f'kytos/mef_eline.{name}'
     event = KytosEvent(name=event_name, content=kwargs)
     controller.buffers.app.put(event)
+
+
+def compare_endpoint_trace(endpoint, vlan, trace):
+    """Compare and endpoint with a trace step."""
+    return endpoint.switch.dpid == trace['dpid']\
+        and endpoint.port_number == trace['port']\
+        and vlan == trace['vlan']
```

Comment:
Can you please add a unit test for this, @ajoaoff ? Without this unit test, the coverage dropped from 80% to 79% (total).